samples/client/petstore/typescript-fetch/builds/default/apis/PetApi.ts

```diff
@@ -24,7 +24,7 @@ import {
 } from '../models';
 
 export interface AddPetRequest {
-    body: Pet;
+    pet: Pet;
```

Comment:
This seems odd, but I don't know why this would've/could've changed. Any suggestions?

Reply:
This is because I accidentally committed the openapi3 samples. Resolved.